### ISSUE TITLE
how this script used ?

### ISSUE THREAD

**Issue by seghier** [rferrazz/pyqt4topyqt5#18]
hi
can you explain step by step how this script used ?

**Comment by Ronkiro** [comment]
It's located at README.md .

> Usage
usage: pyqt4topyqt5.py [-h] [--nosubdir] [--followlinks] [-o O]
                       [--diff [DIFF]] [--diffs] [--nolog] [--nopyqt5]
                       path
Basic example: porting the content of pyqt4app to pyqt5 in the directory pyqt5app:
pyqt4topyqt5.py pyqt4app -o pyqt5app


Basically, you look for a module that uses PyQt4, and then point this script to this module.

python -m pyqt4topyqt5.py pyqt4app

But if you want it with another output, use -o option

python -m pyqt4topyqt5.py pyqt4app -o pyqt5name

**Comment by Ronkiro** [comment]
Yeah, i think the description would be better if they included "python -m", like that:

`python -m pyqt4topyqt5.py pyqt4app -o pyqt5name`

Still, the syntax is kinda easy to use. If you find any errors, opening an new issue would be helpful.

**Comment by tickman07** [comment]
hi, i have the same issue : i tryed different command line, after placing the script in the same folder with source for V4QT programme.

C:\> python -m pyqt4topyqt5.py pyqt4app -o pyqt5name
C:\> python -m pyqt4topyqt5 pyqt4app -o pyqt5name
C:\> python pyqt4topyqt5.py pyqt4app.py -o pyqt5name.py
always i face the same error message : 
pyqt4topyqt5.py: error: unrecognized arguments: pyqt4app
